src/models/SQLCalls.java

```diff
@@ -0,0 +1,94 @@
+package models;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+
+public class SQLCalls {

```

Comment:
I like the idea here. This saves us a lot  of the boilerplate code in Jack's project